src/ServiceInsight/GlobalSuppressions.cs

```diff
@@ -8,4 +8,8 @@
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.LayoutRules", "SA1516:Elements must be separated by blank line", Justification = "Generated File", Scope = "member", Target = "~P:ReleaseDateAttribute.Date")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.LayoutRules", "SA1517:CodeMustNotContainBlankLinesAtStartOfFile", Justification = "Generated File. Bring back when updated to GitVersionTask v3.x")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.LayoutRules", "SA1518:CodeMustNotContainBlankLinesAtEndOfFile", Justification = "Generated File. Bring back when updated to GitVersionTask v3.x")]
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Code", "PCR0001:Await used without specifying ConfigureAwait", Justification = "Default in WPF is fine.")]
\ No newline at end of file
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Code", "PCR0001:Await used without specifying ConfigureAwait", Justification = "Default in WPF is fine.")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1128:ConstructorInitializerMustBeOnOwnLine", Justification = "Generated File.")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.LayoutRules", "SA1516:ElementsMustBeSeparatedByBlankLine", Justification = "Generated File.")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.Naming Rules", "SA1303:ConstFieldNamesMustBeginWithUpperCaseLetter", Justification = "Generated File.")]
```

Comment:
These lines have been added to suppress the StyleCop violations found in the source files added by the new licensing package.

We should not be doing this here because this will suppress those rules for _all_ files.

I'm not sure if StyleCop.Analyzers has a way to suppress specific files from analysis (the best I could find was [this issue which I commented on way back](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/2195)), but I have raised https://github.com/Particular/Operations.Licensing/pull/37 to fix these violations in the source package, so if that PR is merged then we can update to the latest package and remove these lines.